src/chronometer.js

```diff
@@ -1,33 +1,51 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (printTimeCallback) {
+        printTimeCallback();
+      }
+    }, 1000);
   }
 
   getMinutes() {
     // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
 
   getSeconds() {
     // ... your code goes here
+    return Math.floor(this.currentTime % 60);
   }
 
   computeTwoDigitNumber(value) {
     // ... your code goes here
+    if (value < 10) {
+      return `0${value}`;
+    }
+    return `${value}`
   }
 
```

Comment:
Nesse método, o lab pede para transformar os valores em strings, a sugestão que eu dou é usar o .toString( ) primeiro de tudo: value.toString( ); 
Aí vc faz as condições e validações...se for identico a 0 ; retorna '00', 
se for maior ou igual a 10; retorna o valor(em string); senao, retorna '0' + o valor em string 
Vamos fazer esse teste...